README.md

```diff
@@ -4,14 +4,23 @@ This project is Fotogjenens own custom gui library. It is exported as an NPM lib
 
 This project uses React Typescript to write components and Storybook as component documentation.
 
-## Local setup
+## Important file extenstion naming
+* css modules MUST be named `<component-name>.module.css`
+* The projects src/ folder CANNOT contain ANY .js files 
```

Comment:
Kunne du forklart veldig kort hvorfor det ikke kan være .js files?

Reply:
For å få type declarations autogenerert av typescript compileren måtte jeg legge til et felt som ikke godtar at noe javascript er der når den skal compile.